src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBannerCircular.vue

```diff
@@ -20,12 +20,14 @@
 							:rotate="36"
 							:show-number="false"
 						/>
-						<img
+						<kv-contentful-img
 							v-if="imageUrl"
 							class="indicator__image"
-							:src="imageUrl"
-							alt=""
-						>
+							:contentful-src="imageUrl"
+							alt="Promotional banner indicator image"
```

Comment:
Since we're unsure what this image will be, and the headline will announce on line 39, I think it's better to have screenreaders ignore this image by setting `alt=""`